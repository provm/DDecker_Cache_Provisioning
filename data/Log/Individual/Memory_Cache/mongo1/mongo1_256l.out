========== START: Container mongo Memory STATS ==============
cache 62443520
rss 37527552
rss_huge 0
mapped_file 36761600
writeback 0
pgpgin 121151
pgpgout 96744
pgfault 184623
pgmajfault 488
inactive_anon 49152
active_anon 37502976
inactive_file 47767552
active_file 14598144
unevictable 4096
hierarchical_memory_limit 9223372036854771712
total_cache 62443520
total_rss 37527552
total_rss_huge 0
total_mapped_file 36761600
total_writeback 0
total_pgpgin 121151
total_pgpgout 96744
total_pgfault 184623
total_pgmajfault 488
total_inactive_anon 49152
total_active_anon 37502976
total_inactive_file 47767552
total_active_file 14598144
total_unevictable 4096
======== END Container STATS ======
java -cp /home/ubuntu/ycsb_mongo/mongodb-binding/conf:/home/ubuntu/ycsb_mongo/conf:/home/ubuntu/ycsb_mongo/lib/jackson-mapper-asl-1.9.4.jar:/home/ubuntu/ycsb_mongo/lib/jackson-core-asl-1.9.4.jar:/home/ubuntu/ycsb_mongo/lib/htrace-core4-4.1.0-incubating.jar:/home/ubuntu/ycsb_mongo/lib/core-0.12.0.jar:/home/ubuntu/ycsb_mongo/lib/HdrHistogram-2.1.4.jar:/home/ubuntu/ycsb_mongo/mongodb-binding/lib/mongo-java-driver-3.0.3.jar:/home/ubuntu/ycsb_mongo/mongodb-binding/lib/logback-core-1.1.2.jar:/home/ubuntu/ycsb_mongo/mongodb-binding/lib/mongodb-binding-0.12.0.jar:/home/ubuntu/ycsb_mongo/mongodb-binding/lib/logback-classic-1.1.2.jar:/home/ubuntu/ycsb_mongo/mongodb-binding/lib/mongodb-async-driver-2.0.1.jar:/home/ubuntu/ycsb_mongo/mongodb-binding/lib/slf4j-api-1.6.4.jar com.yahoo.ycsb.Client -db com.yahoo.ycsb.db.MongoDbClient -s -P /home/ubuntu/ycsb_mongo/workloads/workload1 -p mongodb.url=mongodb://10.0.3.220:27017/ycsb?w=0 -p maxexecutiontime=600 -t
YCSB Client 0.12.0
Command line: -db com.yahoo.ycsb.db.MongoDbClient -s -P /home/ubuntu/ycsb_mongo/workloads/workload1 -p mongodb.url=mongodb://10.0.3.220:27017/ycsb?w=0 -p maxexecutiontime=600 -t
Loading workload...
Starting test.
Maximum execution time specified as: 600 secs
2017-05-23 23:21:00:275 0 sec: 0 operations; est completion in 0 seconds 
mongo client connection created with mongodb://10.0.3.220:27017/ycsb?w=0
DBWrapper: report latency for each error is false and specific error codes to track for latency are: []
2017-05-23 23:21:09:833 10 sec: 386 operations; 38.6 current ops/sec; est completion in 2 days 23 hours [READ: Count=386, Max=137343, Min=1412, Avg=23388.99, 90=34783, 99=58175, 99.9=137343, 99.99=137343] 
2017-05-23 23:21:19:833 20 sec: 1020 operations; 63.4 current ops/sec; est completion in 2 days 6 hours [READ: Count=634, Max=64959, Min=328, Avg=15738.33, 90=27215, 99=39839, 99.9=47583, 99.99=64959] 
2017-05-23 23:21:29:833 30 sec: 1891 operations; 87.1 current ops/sec; est completion in 1 days 20 hours [READ: Count=871, Max=71935, Min=283, Avg=11429.7, 90=20767, 99=36031, 99.9=58143, 99.99=71935] 
2017-05-23 23:21:39:832 40 sec: 2969 operations; 107.8 current ops/sec; est completion in 1 days 13 hours [READ: Count=1078, Max=43327, Min=233, Avg=9261.99, 90=13959, 99=28335, 99.9=42303, 99.99=43327] 
2017-05-23 23:21:49:833 50 sec: 4084 operations; 111.5 current ops/sec; est completion in 1 days 9 hours [READ: Count=1115, Max=43423, Min=222, Avg=8957.57, 90=13991, 99=26991, 99.9=41471, 99.99=43423] 
2017-05-23 23:21:59:833 60 sec: 5218 operations; 113.4 current ops/sec; est completion in 1 days 7 hours [READ: Count=1134, Max=45151, Min=193, Avg=8804.85, 90=13935, 99=35967, 99.9=42943, 99.99=45151] 
2017-05-23 23:22:09:833 70 sec: 6435 operations; 121.7 current ops/sec; est completion in 1 days 6 hours [READ: Count=1217, Max=42271, Min=173, Avg=8200.12, 90=13183, 99=16343, 99.9=42239, 99.99=42271] 
2017-05-23 23:22:19:833 80 sec: 7631 operations; 119.6 current ops/sec; est completion in 1 days 5 hours [READ: Count=1196, Max=45887, Min=169, Avg=8352.56, 90=13351, 99=16167, 99.9=38975, 99.99=45887] 
2017-05-23 23:22:29:833 90 sec: 8857 operations; 122.6 current ops/sec; est completion in 1 days 4 hours [READ: Count=1226, Max=39967, Min=166, Avg=8140.34, 90=13239, 99=16335, 99.9=39423, 99.99=39967] 
2017-05-23 23:22:39:833 100 sec: 10129 operations; 127.2 current ops/sec; est completion in 1 days 3 hours [READ: Count=1272, Max=36223, Min=170, Avg=7848.58, 90=13127, 99=16079, 99.9=36223, 99.99=36223] 
2017-05-23 23:22:49:833 110 sec: 11417 operations; 128.8 current ops/sec; est completion in 1 days 2 hours [READ: Count=1288, Max=41791, Min=104, Avg=7757.9, 90=13247, 99=16559, 99.9=39007, 99.99=41791] 
2017-05-23 23:22:59:833 120 sec: 12649 operations; 123.2 current ops/sec; est completion in 1 days 2 hours [READ: Count=1232, Max=44287, Min=101, Avg=8116.34, 90=13151, 99=32959, 99.9=42399, 99.99=44287] 
2017-05-23 23:23:09:833 130 sec: 13944 operations; 129.5 current ops/sec; est completion in 1 days 1 hours [READ: Count=1295, Max=42207, Min=99, Avg=7718.95, 90=13263, 99=18383, 99.9=41471, 99.99=42207] 
2017-05-23 23:23:19:832 140 sec: 15270 operations; 132.6 current ops/sec; est completion in 1 days 1 hours [READ: Count=1326, Max=44063, Min=98, Avg=7536.05, 90=12447, 99=16575, 99.9=41311, 99.99=44063] 
2017-05-23 23:23:29:833 150 sec: 16596 operations; 132.6 current ops/sec; est completion in 1 days 1 hours [READ: Count=1326, Max=64319, Min=97, Avg=7536.62, 90=13143, 99=32223, 99.9=58143, 99.99=64319] 
2017-05-23 23:23:39:833 160 sec: 17933 operations; 133.7 current ops/sec; est completion in 1 days [READ: Count=1337, Max=39039, Min=97, Avg=7472.7, 90=13047, 99=16119, 99.9=37759, 99.99=39039] 
2017-05-23 23:23:49:833 170 sec: 19332 operations; 139.9 current ops/sec; est completion in 1 days [READ: Count=1399, Max=42655, Min=94, Avg=7151.21, 90=12735, 99=16655, 99.9=38015, 99.99=42655] 
2017-05-23 23:23:59:833 180 sec: 20725 operations; 139.3 current ops/sec; est completion in 1 days [READ: Count=1393, Max=41535, Min=94, Avg=7179.46, 90=13447, 99=19183, 99.9=37567, 99.99=41535] 
2017-05-23 23:24:09:833 190 sec: 22173 operations; 144.8 current ops/sec; est completion in 23 hours 45 minutes [READ: Count=1448, Max=57311, Min=94, Avg=6898.81, 90=12999, 99=18079, 99.9=42431, 99.99=57311] 
2017-05-23 23:24:19:833 200 sec: 23673 operations; 150 current ops/sec; est completion in 23 hours 24 minutes [READ: Count=1500, Max=41919, Min=93, Avg=6660.28, 90=12735, 99=17695, 99.9=35583, 99.99=41919] 
2017-05-23 23:24:29:833 210 sec: 25200 operations; 152.7 current ops/sec; est completion in 23 hours 5 minutes [READ: Count=1527, Max=45471, Min=91, Avg=6554.27, 90=12519, 99=18479, 99.9=36447, 99.99=45471] 
2017-05-23 23:24:39:833 220 sec: 26676 operations; 147.6 current ops/sec; est completion in 22 hours 50 minutes [READ: Count=1476, Max=48159, Min=96, Avg=6768.71, 90=12879, 99=18191, 99.9=42399, 99.99=48159] 
2017-05-23 23:24:49:833 230 sec: 28218 operations; 154.2 current ops/sec; est completion in 22 hours 34 minutes [READ: Count=1542, Max=41311, Min=91, Avg=6483.13, 90=12591, 99=17695, 99.9=37055, 99.99=41311] 
2017-05-23 23:24:59:833 240 sec: 29770 operations; 155.2 current ops/sec; est completion in 22 hours 19 minutes [READ: Count=1552, Max=53279, Min=93, Avg=6441.22, 90=12599, 99=17919, 99.9=43327, 99.99=53279] 
2017-05-23 23:25:09:833 250 sec: 31341 operations; 157.1 current ops/sec; est completion in 22 hours 5 minutes [READ: Count=1571, Max=42879, Min=92, Avg=6358.21, 90=12847, 99=18671, 99.9=35743, 99.99=42879] 
2017-05-23 23:25:19:833 260 sec: 32943 operations; 160.2 current ops/sec; est completion in 21 hours 51 minutes [READ: Count=1602, Max=69439, Min=89, Avg=6245.26, 90=12687, 99=20655, 99.9=43167, 99.99=69439] 
2017-05-23 23:25:29:833 270 sec: 34627 operations; 168.4 current ops/sec; est completion in 21 hours 35 minutes [READ: Count=1684, Max=48863, Min=92, Avg=5933.26, 90=12799, 99=17935, 99.9=35711, 99.99=48863] 
2017-05-23 23:25:39:833 280 sec: 36260 operations; 163.3 current ops/sec; est completion in 21 hours 22 minutes [READ: Count=1633, Max=47871, Min=97, Avg=6124.36, 90=13207, 99=19375, 99.9=37727, 99.99=47871] 
2017-05-23 23:25:49:833 290 sec: 37956 operations; 169.6 current ops/sec; est completion in 21 hours 8 minutes [READ: Count=1696, Max=42079, Min=95, Avg=5893.42, 90=12767, 99=19695, 99.9=39487, 99.99=42079] 
2017-05-23 23:25:59:833 300 sec: 39699 operations; 174.3 current ops/sec; est completion in 20 hours 54 minutes [READ: Count=1743, Max=38911, Min=95, Avg=5733.21, 90=12431, 99=18543, 99.9=38431, 99.99=38911] 
2017-05-23 23:26:09:833 310 sec: 41486 operations; 178.7 current ops/sec; est completion in 20 hours 40 minutes [READ: Count=1787, Max=41535, Min=91, Avg=5594.33, 90=12287, 99=18655, 99.9=37119, 99.99=41535] 
2017-05-23 23:26:19:833 320 sec: 43227 operations; 174.1 current ops/sec; est completion in 20 hours 28 minutes [READ: Count=1741, Max=46111, Min=92, Avg=5732.08, 90=13071, 99=21023, 99.9=42303, 99.99=46111] 
2017-05-23 23:26:29:832 330 sec: 44948 operations; 172.1 current ops/sec; est completion in 20 hours 18 minutes [READ: Count=1721, Max=41407, Min=93, Avg=5812.66, 90=12503, 99=20495, 99.9=34655, 99.99=41407] 
2017-05-23 23:26:39:833 340 sec: 46692 operations; 174.4 current ops/sec; est completion in 20 hours 7 minutes [READ: Count=1744, Max=42463, Min=91, Avg=5732.88, 90=12479, 99=22303, 99.9=39391, 99.99=42463] 
2017-05-23 23:26:49:833 350 sec: 48505 operations; 181.3 current ops/sec; est completion in 19 hours 56 minutes [READ: Count=1813, Max=45887, Min=93, Avg=5516.48, 90=12063, 99=18863, 99.9=37023, 99.99=45887] 
2017-05-23 23:26:59:833 360 sec: 50258 operations; 175.3 current ops/sec; est completion in 19 hours 47 minutes [READ: Count=1753, Max=44095, Min=92, Avg=5698.02, 90=12775, 99=18927, 99.9=37375, 99.99=44095] 
2017-05-23 23:27:09:833 370 sec: 52050 operations; 179.2 current ops/sec; est completion in 19 hours 38 minutes [READ: Count=1792, Max=41599, Min=91, Avg=5581.15, 90=12071, 99=19119, 99.9=37695, 99.99=41599] 
2017-05-23 23:27:19:833 380 sec: 53845 operations; 179.5 current ops/sec; est completion in 19 hours 29 minutes [READ: Count=1795, Max=43359, Min=92, Avg=5563.65, 90=12167, 99=19391, 99.9=41119, 99.99=43359] 
2017-05-23 23:27:29:833 390 sec: 55697 operations; 185.2 current ops/sec; est completion in 19 hours 20 minutes [READ: Count=1852, Max=39103, Min=91, Avg=5399.54, 90=12375, 99=17759, 99.9=37343, 99.99=39103] 
2017-05-23 23:27:39:833 400 sec: 57431 operations; 173.4 current ops/sec; est completion in 19 hours 14 minutes [READ: Count=1734, Max=46623, Min=94, Avg=5764.08, 90=12935, 99=20527, 99.9=40639, 99.99=46623] 
2017-05-23 23:27:49:833 410 sec: 59227 operations; 179.6 current ops/sec; est completion in 19 hours 6 minutes [READ: Count=1796, Max=49855, Min=91, Avg=5566.73, 90=12567, 99=19663, 99.9=44127, 99.99=49855] 
2017-05-23 23:27:59:833 420 sec: 60989 operations; 176.2 current ops/sec; est completion in 19 hours [READ: Count=1762, Max=42079, Min=91, Avg=5673.6, 90=12695, 99=19775, 99.9=39455, 99.99=42079] 
2017-05-23 23:28:09:833 430 sec: 62762 operations; 177.3 current ops/sec; est completion in 18 hours 54 minutes [READ: Count=1773, Max=52159, Min=92, Avg=5636.56, 90=12607, 99=19535, 99.9=38175, 99.99=52159] 
2017-05-23 23:28:19:833 440 sec: 64509 operations; 174.7 current ops/sec; est completion in 18 hours 49 minutes [READ: Count=1747, Max=43231, Min=91, Avg=5720.42, 90=12503, 99=20287, 99.9=41087, 99.99=43231] 
2017-05-23 23:28:29:833 450 sec: 66265 operations; 175.6 current ops/sec; est completion in 18 hours 44 minutes [READ: Count=1756, Max=38623, Min=94, Avg=5692.44, 90=12487, 99=18495, 99.9=33215, 99.99=38623] 
2017-05-23 23:28:39:832 460 sec: 68092 operations; 182.7 current ops/sec; est completion in 18 hours 38 minutes [READ: Count=1827, Max=43711, Min=91, Avg=5473.83, 90=12207, 99=20623, 99.9=37535, 99.99=43711] 
2017-05-23 23:28:49:833 470 sec: 69839 operations; 174.7 current ops/sec; est completion in 18 hours 33 minutes [READ: Count=1747, Max=43135, Min=92, Avg=5716.08, 90=12207, 99=20031, 99.9=34879, 99.99=43135] 
2017-05-23 23:28:59:833 480 sec: 71714 operations; 187.5 current ops/sec; est completion in 18 hours 27 minutes [READ: Count=1875, Max=39359, Min=93, Avg=5334.34, 90=11967, 99=19231, 99.9=36863, 99.99=39359] 
2017-05-23 23:29:09:833 490 sec: 73473 operations; 175.9 current ops/sec; est completion in 18 hours 23 minutes [READ: Count=1759, Max=41919, Min=93, Avg=5682.87, 90=12655, 99=18959, 99.9=37695, 99.99=41919] 
2017-05-23 23:29:19:833 500 sec: 75282 operations; 180.9 current ops/sec; est completion in 18 hours 18 minutes [READ: Count=1809, Max=44287, Min=91, Avg=5521.96, 90=12567, 99=17711, 99.9=38079, 99.99=44287] 
2017-05-23 23:29:29:833 510 sec: 77109 operations; 182.7 current ops/sec; est completion in 18 hours 13 minutes [READ: Count=1827, Max=47519, Min=93, Avg=5469.7, 90=11911, 99=18447, 99.9=34463, 99.99=47519] 
2017-05-23 23:29:39:833 520 sec: 78949 operations; 184 current ops/sec; est completion in 18 hours 9 minutes [READ: Count=1840, Max=41887, Min=91, Avg=5436.46, 90=12343, 99=20063, 99.9=37087, 99.99=41887] 
2017-05-23 23:29:49:833 530 sec: 80679 operations; 173 current ops/sec; est completion in 18 hours 6 minutes [READ: Count=1730, Max=40863, Min=93, Avg=5775.22, 90=12703, 99=19407, 99.9=37503, 99.99=40863] 
2017-05-23 23:29:59:833 540 sec: 82520 operations; 184.1 current ops/sec; est completion in 18 hours 1 minutes [READ: Count=1841, Max=43711, Min=91, Avg=5423.76, 90=12095, 99=19455, 99.9=37951, 99.99=43711] 
2017-05-23 23:30:09:833 550 sec: 84260 operations; 174 current ops/sec; est completion in 17 hours 58 minutes [READ: Count=1740, Max=41887, Min=92, Avg=5753.57, 90=12559, 99=20927, 99.9=37855, 99.99=41887] 
2017-05-23 23:30:19:832 560 sec: 86030 operations; 177 current ops/sec; est completion in 17 hours 55 minutes [READ: Count=1770, Max=42751, Min=91, Avg=5647.46, 90=12527, 99=20815, 99.9=35487, 99.99=42751] 
2017-05-23 23:30:29:833 570 sec: 87816 operations; 178.6 current ops/sec; est completion in 17 hours 52 minutes [READ: Count=1786, Max=45151, Min=93, Avg=5595.75, 90=12431, 99=19215, 99.9=38655, 99.99=45151] 
2017-05-23 23:30:39:833 580 sec: 89498 operations; 168.2 current ops/sec; est completion in 17 hours 50 minutes [READ: Count=1682, Max=43935, Min=93, Avg=5942.13, 90=12703, 99=19279, 99.9=39967, 99.99=43935] 
2017-05-23 23:30:49:833 590 sec: 91270 operations; 177.2 current ops/sec; est completion in 17 hours 47 minutes [READ: Count=1772, Max=42815, Min=92, Avg=5639.4, 90=12359, 99=19839, 99.9=40799, 99.99=42815] 
2017-05-23 23:30:59:833 600 sec: 92964 operations; 169.4 current ops/sec; est completion in 17 hours 45 minutes [READ: Count=1694, Max=173951, Min=90, Avg=5902.52, 90=12695, 99=19535, 99.9=40127, 99.99=173951] 
Maximum time elapsed. Requesting stop for the workload.
Stop requested for workload. Now Joining!
2017-05-23 23:30:59:854 600 sec: 92966 operations; 90.91 current ops/sec; est completion in 17 hours 45 minutes [CLEANUP: Count=1, Max=1130, Min=1130, Avg=1130, 90=1130, 99=1130, 99.9=1130, 99.99=1130] [READ: Count=2, Max=10519, Min=9616, Avg=10068, 90=10519, 99=10519, 99.9=10519, 99.99=10519] 
[OVERALL], RunTime(ms), 600022.0
[OVERALL], Throughput(ops/sec), 154.93765228608285
[TOTAL_GCS_PS_Scavenge], Count, 154.0
[TOTAL_GC_TIME_PS_Scavenge], Time(ms), 186.0
[TOTAL_GC_TIME_%_PS_Scavenge], Time(%), 0.030998863375009584
[TOTAL_GCS_PS_MarkSweep], Count, 0.0
[TOTAL_GC_TIME_PS_MarkSweep], Time(ms), 0.0
[TOTAL_GC_TIME_%_PS_MarkSweep], Time(%), 0.0
[TOTAL_GCs], Count, 154.0
[TOTAL_GC_TIME], Time(ms), 186.0
[TOTAL_GC_TIME_%], Time(%), 0.030998863375009584
[CLEANUP], Operations, 1.0
[CLEANUP], AverageLatency(us), 1130.0
[CLEANUP], MinLatency(us), 1130.0
[CLEANUP], MaxLatency(us), 1130.0
[CLEANUP], 95thPercentileLatency(us), 1130.0
[CLEANUP], 99thPercentileLatency(us), 1130.0
[READ], Operations, 92966.0
[READ], AverageLatency(us), 6439.475410365079
[READ], MinLatency(us), 89.0
[READ], MaxLatency(us), 173951.0
[READ], 95thPercentileLatency(us), 15103.0
[READ], 99thPercentileLatency(us), 22639.0
[READ], Return=OK, 92966
========== STOP: Container mongo Memory STATS ==============
cache 744374272
rss 255393792
rss_huge 0
mapped_file 21458944
writeback 0
pgpgin 735996
pgpgout 491912
pgfault 242365
pgmajfault 518
inactive_anon 128753664
active_anon 126709760
inactive_file 372166656
active_file 372133888
unevictable 4096
hierarchical_memory_limit 999997440
total_cache 744374272
total_rss 255393792
total_rss_huge 0
total_mapped_file 21458944
total_writeback 0
total_pgpgin 735996
total_pgpgout 491912
total_pgfault 242365
total_pgmajfault 518
total_inactive_anon 128753664
total_active_anon 126709760
total_inactive_file 372166656
total_active_file 372133888
total_unevictable 4096
======== END Container STATS ======
