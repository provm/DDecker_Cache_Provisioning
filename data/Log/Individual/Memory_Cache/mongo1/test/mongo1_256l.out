========== START: Container mongo Memory STATS ==============
cache 62816256
rss 37457920
rss_huge 0
mapped_file 36761600
writeback 0
pgpgin 120920
pgpgout 96439
pgfault 184649
pgmajfault 487
inactive_anon 49152
active_anon 37453824
inactive_file 47738880
active_file 14999552
unevictable 4096
hierarchical_memory_limit 9223372036854771712
total_cache 62816256
total_rss 37457920
total_rss_huge 0
total_mapped_file 36761600
total_writeback 0
total_pgpgin 120920
total_pgpgout 96439
total_pgfault 184649
total_pgmajfault 487
total_inactive_anon 49152
total_active_anon 37453824
total_inactive_file 47738880
total_active_file 14999552
total_unevictable 4096
======== END Container STATS ======
java -cp /home/ubuntu/ycsb_mongo/mongodb-binding/conf:/home/ubuntu/ycsb_mongo/conf:/home/ubuntu/ycsb_mongo/lib/jackson-mapper-asl-1.9.4.jar:/home/ubuntu/ycsb_mongo/lib/jackson-core-asl-1.9.4.jar:/home/ubuntu/ycsb_mongo/lib/htrace-core4-4.1.0-incubating.jar:/home/ubuntu/ycsb_mongo/lib/core-0.12.0.jar:/home/ubuntu/ycsb_mongo/lib/HdrHistogram-2.1.4.jar:/home/ubuntu/ycsb_mongo/mongodb-binding/lib/mongo-java-driver-3.0.3.jar:/home/ubuntu/ycsb_mongo/mongodb-binding/lib/logback-core-1.1.2.jar:/home/ubuntu/ycsb_mongo/mongodb-binding/lib/mongodb-binding-0.12.0.jar:/home/ubuntu/ycsb_mongo/mongodb-binding/lib/logback-classic-1.1.2.jar:/home/ubuntu/ycsb_mongo/mongodb-binding/lib/mongodb-async-driver-2.0.1.jar:/home/ubuntu/ycsb_mongo/mongodb-binding/lib/slf4j-api-1.6.4.jar com.yahoo.ycsb.Client -db com.yahoo.ycsb.db.MongoDbClient -s -P /home/ubuntu/ycsb_mongo/workloads/workload1 -p mongodb.url=mongodb://10.0.3.220:27017/ycsb?w=0 -p maxexecutiontime=600 -load
YCSB Client 0.12.0
Command line: -db com.yahoo.ycsb.db.MongoDbClient -s -P /home/ubuntu/ycsb_mongo/workloads/workload1 -p mongodb.url=mongodb://10.0.3.220:27017/ycsb?w=0 -p maxexecutiontime=600 -load
Loading workload...
Starting test.
Maximum execution time specified as: 600 secs
2017-05-21 18:09:20:555 0 sec: 0 operations; est completion in 0 seconds 
mongo client connection created with mongodb://10.0.3.220:27017/ycsb?w=0
DBWrapper: report latency for each error is false and specific error codes to track for latency are: []
2017-05-21 18:09:30:091 10 sec: 1907 operations; 190.7 current ops/sec; est completion in 1 hours 27 minutes [INSERT: Count=1907, Max=973823, Min=178, Avg=4601.58, 90=684, 99=3083, 99.9=790527, 99.99=973823] 
2017-05-21 18:09:40:091 20 sec: 4896 operations; 298.9 current ops/sec; est completion in 1 hours 7 minutes [INSERT: Count=2989, Max=1003519, Min=104, Avg=3018.31, 90=219, 99=562, 99.9=777215, 99.99=1003519] 
2017-05-21 18:09:50:091 30 sec: 8236 operations; 334 current ops/sec; est completion in 1 hours [INSERT: Count=3340, Max=1248255, Min=78, Avg=3214.52, 90=101, 99=162, 99.9=951295, 99.99=1248255] 
2017-05-21 18:10:00:091 40 sec: 12366 operations; 413 current ops/sec; est completion in 53 minutes [INSERT: Count=4130, Max=1241087, Min=35, Avg=2455.26, 90=81, 99=334, 99.9=809471, 99.99=1241087] 
2017-05-21 18:10:10:091 50 sec: 116167 operations; 10380.1 current ops/sec; est completion in 6 minutes [INSERT: Count=103806, Max=474623, Min=33, Avg=93.06, 90=37, 99=60, 99.9=1084, 99.99=227583] 
2017-05-21 18:10:20:091 60 sec: 282346 operations; 16617.9 current ops/sec; est completion in 2 minutes [INSERT: Count=166180, Max=2447, Min=33, Avg=56.37, 90=60, 99=97, 99.9=110, 99.99=1221] 
2017-05-21 18:10:30:091 70 sec: 419956 operations; 13761 current ops/sec; est completion in 1 minutes [INSERT: Count=137607, Max=149503, Min=33, Avg=68.26, 90=97, 99=107, 99.9=124, 99.99=1314] 
2017-05-21 18:10:40:091 80 sec: 584605 operations; 16464.9 current ops/sec; est completion in 57 seconds [INSERT: Count=164648, Max=3397, Min=50, Avg=56.79, 90=60, 99=101, 99.9=115, 99.99=1232] 
2017-05-21 18:10:50:091 90 sec: 751546 operations; 16694.1 current ops/sec; est completion in 30 seconds [INSERT: Count=166939, Max=4311, Min=50, Avg=56.05, 90=60, 99=92, 99.9=107, 99.99=1240] 
2017-05-21 18:11:00:091 100 sec: 918929 operations; 16738.3 current ops/sec; est completion in 9 seconds [INSERT: Count=167385, Max=2321, Min=49, Avg=55.79, 90=60, 99=92, 99.9=108, 99.99=1171] 
Could not wait until max specified time, TerminatorThread interrupted.
2017-05-21 18:11:06:261 106 sec: 1000000 operations; 13139.55 current ops/sec; [CLEANUP: Count=1, Max=1128, Min=1128, Avg=1128, 90=1128, 99=1128, 99.9=1128, 99.99=1128] [INSERT: Count=81069, Max=2451, Min=49, Avg=71.37, 90=101, 99=108, 99.9=130, 99.99=1247] 
[OVERALL], RunTime(ms), 106170.0
[OVERALL], Throughput(ops/sec), 9418.856550814731
[TOTAL_GCS_PS_Scavenge], Count, 485.0
[TOTAL_GC_TIME_PS_Scavenge], Time(ms), 539.0
[TOTAL_GC_TIME_%_PS_Scavenge], Time(%), 0.507676368088914
[TOTAL_GCS_PS_MarkSweep], Count, 0.0
[TOTAL_GC_TIME_PS_MarkSweep], Time(ms), 0.0
[TOTAL_GC_TIME_%_PS_MarkSweep], Time(%), 0.0
[TOTAL_GCs], Count, 485.0
[TOTAL_GC_TIME], Time(ms), 539.0
[TOTAL_GC_TIME_%], Time(%), 0.507676368088914
[CLEANUP], Operations, 1.0
[CLEANUP], AverageLatency(us), 1128.0
[CLEANUP], MinLatency(us), 1128.0
[CLEANUP], MaxLatency(us), 1128.0
[CLEANUP], 95thPercentileLatency(us), 1128.0
[CLEANUP], 99thPercentileLatency(us), 1128.0
[INSERT], Operations, 1000000.0
[INSERT], AverageLatency(us), 100.925723
[INSERT], MinLatency(us), 33.0
[INSERT], MaxLatency(us), 1248255.0
[INSERT], 95thPercentileLatency(us), 96.0
[INSERT], 99thPercentileLatency(us), 105.0
[INSERT], Return=OK, 1000000
========== STOP: Container mongo Memory STATS ==============
cache 160321536
rss 460836864
rss_huge 0
mapped_file 37576704
writeback 0
pgpgin 250139
pgpgout 98489
pgfault 290718
pgmajfault 499
inactive_anon 45056
active_anon 460861440
inactive_file 144449536
active_file 15798272
unevictable 4096
hierarchical_memory_limit 999997440
total_cache 160321536
total_rss 460836864
total_rss_huge 0
total_mapped_file 37576704
total_writeback 0
total_pgpgin 250139
total_pgpgout 98489
total_pgfault 290718
total_pgmajfault 499
total_inactive_anon 45056
total_active_anon 460861440
total_inactive_file 144449536
total_active_file 15798272
total_unevictable 4096
======== END Container STATS ======
